cucumber/src/main/java/org/chodavarapu/datamill/cucumber/CommandLineSteps.java

```diff
@@ -176,6 +192,27 @@ public void verifyTemporaryDirectoryHasFiles(List<String> names) throws Exceptio
         }
     }
 
+    @Then("^the result should not contain \"(.+)\"$")
```

Comment:
Minor: maybe call "the result" in these "standard output" or "command output" just to be a bit more precise